### PR TITLE
feat(navigation): adding patient navbar and multipages

### PR DESCRIPTION
**Pull request by SpiritGY**
# Description

adding mulitpage and new navbar for doctor app

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

switch navbar to new version accurate to patient app one
adding multipage navigation for doctoe app

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to a clickup task
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

**Review by lucas-louis — CHANGES_REQUESTED** [review]
Navbar:
- It's a bit complicated to click on a tab in the navbar
- The icon has the good size?

"Mes services"
- Uppercase for patients and diagnotics texts
- All the card needs to be clickable and not only the text/ icon

File: medecin_mobile/lib/screens/dashboard/services.dart [hunk 1]
```diff
@@ -109,7 +109,10 @@ class BarItem extends StatelessWidget {
         padding: const EdgeInsets.all(8),
         child: GestureDetector(
           onTap: ontap,
-          child: Row(
+          child:Container(
```
Comment (lucas-louis) [inline medecin_mobile/lib/screens/dashboard/services.dart:112]: Not linted

**Review by lucas-louis — CHANGES_REQUESTED** [review]
Navbar:
- The profile icon is too small
- Can you change the size of the text and use the small size instead?

Services:
- The background doesn't take all the card

**Review by lucas-louis — CHANGES_REQUESTED** [review]
Services:
- The background of each tab doesn't take all the space available so there is a small white background that doesn't take all the available space of the tab
image

**Review by MarvinFlm — APPROVED** [review]
Good

**Pull request merged**